src/web/app/pages-instructor/instructor-home-page/instructor-home-page.component.ts

```diff
@@ -151,7 +145,7 @@ export class InstructorHomePageComponent extends InstructorSessionModalPageCompo
     this.courseService.binCourse(courseId).subscribe((course: Course) => {
       this.loadCourses();
       this.statusMessageService.showSuccessMessage(
-        `The course ${course.courseId} has been deleted. You can restore it from the Recycle Bin manually.`);
+          `The course ${course.courseId} has been deleted. You can restore it from the Recycle Bin manually.`);
```

Comment:
Extra indentation here?

Reply:
Continuation Indentation should be 4. The previous indentation is kind of wrong.